content/docs/2.6/scalers/azure-app-insights.md

```diff
@@ -0,0 +1,197 @@
++++
+title = "Azure Application Insights"
+layout = "scaler"
+availability = "v2.6+"
+maintainer = "Community"
+description = "Scale applications based on Azure Application Insights metrics."
+go_file = "azure_app_insights_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `azure-app-insights` trigger that scales based on an Azure Application Insights metric.
+
+```yaml
+triggers:
+- type: azure-app-insights
+  metadata:
```

Comment:
Sanity check - Are we showcasing all parameters here?

Reply:
I believe so.